src/users/users.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { User } from './user.entity';
+
+@Injectable()
+export class UsersService {
+    constructor(
+        @InjectRepository(User)
+        private usersRepository: Repository<User>
+    ){}
+
+    findAll(): Promise<User[]>{
+        return this.usersRepository.find()
+    }
+    findOne(id:string):Promise<User>{
+        return this.usersRepository.findOne(id);
+    }
+   async update(data:any): Promise<User>{
+       return await this.usersRepository.save(data);
+
+   }
+    create(data:any): Promise<User>{
+    return this.usersRepository.save(data);
+    
+}
+
```

Comment:
remember to run `npm run format` before pushing your code :white_circle:

Reply:
thanks :+1: